### ISSUE TITLE
[Bug Bash] The “Open” button is enabled automatically even no config file is selected after reopening the “Configuration Files” dialog

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#12807]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

Dev\6.8.0.60

### Impact

None

#### Repro Steps:
1.Create a project in VS and open the PM UI or PMC window.
2.Click the settings button (gear button) on PM UI or PMC window.
3.Click the NuGet Package Manager->Configuration Files in the opened “Options” dialog.
4.Click a config file in the “Configuration Files” list and close this dialog.
5.Click the settings button (gear button) again and click “Open” button.
#### Expected:
The “Open” button should be disabled because no configuration file is selected when reopening the dialog in step5.
#### Actual:
The “Open” button is enabled automatically and nothing is opened after clicking the “Open” button in step5.
![image](https://github.com/NuGet/Home/assets/20632625/b82c3ce2-0732-4584-a232-7934bd5a98f4)
#### Note:
It's not a regression since it is a new feature.

**Comment by CiciLi1** [comment]
It still reproes on VS Main\35222.175 + NuGet Client Dev\6.12.0.82.

**Comment by donnie-msft** [comment]
Fixed by https://github.com/NuGet/NuGet.Client/pull/6393
Verified in 18.3.0 Insiders 1
